scripts/rspec/rspec-templates/VbNetTestSnippet.cs

```diff
@@ -1,7 +1,5 @@
 ﻿        [TestMethod]
         [TestCategory("Rule")]
-        public void $DiagnosticClassName$_VB()
-        {
+        public void $DiagnosticClassName$_VB() =>
             Verifier.VerifyAnalyzer(@"TestCases\$DiagnosticClassName$.vb",
-                new SonarAnalyzer.Rules.VisualBasic.$DiagnosticClassName$());
-        }
\ No newline at end of file
+                new SonarAnalyzer.Rules.VisualBasic.$DiagnosticClassName$());
```

Comment:
New imports snippet will be needed
```suggestion
                new VB.$DiagnosticClassName$());
```